src/lkl/virtio_netdev.c

```diff
@@ -70,6 +70,16 @@ static void lkl_deliver_irq(uint64_t dev_id)
 int lkl_virtio_netdev_add(struct virtio_dev* netdev)
 {
     int ret = -1;
+
+    if (!netdev)
+        return -1;
+
+    if (!oe_is_outside_enclave(netdev, sizeof(struct virtio_dev)))
+        oe_abort();
+
+    // TODO: Ensure netdev->queue/ops/config_data/base are outside enclave on
```

Comment:
@prp I assume we will have to check queue/ops/config_data/base wherever they are accessed. If that's not so, please let me know so I don't go down unnecessary rabbit holes :)